umpost/conversion_driver_esm1p5.py

```diff
@@ -110,14 +122,73 @@ def find_matching_fields_files(dir_contents, fields_file_name_pattern):
     return fields_file_paths
 
 
-def convert_fields_file_list(fields_file_paths, nc_write_dir):
+def get_nc_filename(fields_file_name, date=None):
+    """
+    Format a netCDF output filename based on the input fields file name and
+    its date. Assumes fields_file_name follows ESM1.5's naming convention
+    '{5 char run_id}.pa{unit}{date encoding}`.
+
+    Parameters
+    ----------
+    fields_file_name: name of fields file to be converted.
+    date: tuple of form (year, month, day) associated with fields file data,
+          or None. If None, ".nc" will concatenated to the original fields
```

Comment:
There's a small typo with "concatenated".

Reply:
Good catch! Fixed up in 797a824